### PR TITLE
resource_control: ru consistent with serverless 

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Refine descriptions of RU to make it consistent with TiDB Serverless

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/14033
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

File: tidb-resource-control.md [hunk 1]
```diff
@@ -43,21 +43,46 @@ Currently, the resource control feature has the following limitations:
...
-| Basic overhead of a write request  | 1.5 RU   |
-
-Based on the above table, assuming that the TiKV time consumed by a resource group is `c` milliseconds, `r1` times of requests read `r2` KB data, `w1` times of write requests write `w2` KB data, and the number of non-witness TiKV nodes in the cluster is `n`. Then, the formula for the total RUs consumed by the resource group is as follows:
-
-`c`\* 1/3 + (`r1` \* 0.25 + `r2` \* 1/64) + (1.5 \* `w1` + `w2` \* 1 \* `n`)
+> **Note:**
+>
+> - Each write operation is eventually replicated to all replicas (by default TiKV has 3 replicas). Each replication operation is considered a different write operation.
+> - In addition to queries executed by the user, RU can be consumed by background tasks, such as automated statistical information collection.
+> - The preceding table lists only the resources involved in computing RUs for on-premises TiDB clusters, excluding the network and storage components. For TiDB Serverless RUs, see [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud- serverless-pricing-details/).
```
Comment (HuSharp) [inline tidb-resource-control.md:85]:
```suggestion
> - The preceding table lists only the resources involved in computing RUs for on-premises TiDB clusters, excluding the network and storage components. For TiDB Serverless RUs, see [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/).
```

**Review by HuSharp — APPROVED** [review]
LGTM

**Review by CabinfeverB — APPROVED** [review]

File: tidb-resource-control.md [hunk 2]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
 
 ## What is Request Unit (RU)
 
-Request Unit (RU) is a unified abstraction unit in TiDB for system resources, which currently includes CPU, IOPS, and IO bandwidth metrics. The consumption of these three metrics is represented by RU according to a certain ratio.
+Request Unit (RU) is a unified abstraction unit in TiDB for system resources, which currently includes CPU, IOPS, and IO bandwidth metrics. It is used to indicate the amount of resources consumed by a single request to the database. The number of RUs consumed by a request depends on a variety of factors, such as the type of operation, and the amount of data being retrieved or modified. Currently, the RU contains statistics for the resources in the following table:
```
Comment (qiancai) [inline tidb-resource-control.md:46]:
```suggestion
Request Unit (RU) is a unified abstraction unit in TiDB for system resources, which currently includes CPU, IOPS, and IO bandwidth metrics. It is used to indicate the amount of resources consumed by a single request to the database. The number of RUs consumed by a request depends on a variety of factors, such as the type of operations, and the amount of data being queried or modified. Currently, the RU contains consumption statistics for the resources in the following table:
```

Comment (qiancai) [reply tidb-resource-control.md:46]: reason: using "consumption statistics" to differentiate it from https://docs.pingcap.com/tidb/dev/statistics

File: tidb-resource-control.md [hunk 3]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
...
+
+<table>
+    <thead>
+        <tr>
+            <th>Resource Type</th>
```
Comment (qiancai) [inline tidb-resource-control.md:51]:
```suggestion
            <th>Resource type</th>
```

File: tidb-resource-control.md [hunk 4]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
...
+            <th>RU Consumption</th>
```
Comment (qiancai) [inline tidb-resource-control.md:52]:
```suggestion
            <th>RU consumption</th>
```

File: tidb-resource-control.md [hunk 5]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
...
+        </tr>
+        <tr>
+            <td>8 storage read requests consume 1 RU</td>
+        </tr>
+        <tr>
+            <td>64 KiB read request payload consumes 1 RU</td>
+        </tr>
+        <tr>
+            <td rowspan="3">Write</td>
+            <td>1 storage write batch consumes 1 RU * (the number of replicas)</td>
```
Comment (qiancai) [inline tidb-resource-control.md:68]:
```suggestion
            <td>1 storage write batch consumes 1 RU for each replica</td>
```

File: tidb-resource-control.md [hunk 6]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
...
-| Write IO        | 1 RU/KB          |
-| Basic overhead of a read request   | 0.25 RU  |
-| Basic overhead of a write request  | 1.5 RU   |
-
-Based on the above table, assuming that the TiKV time consumed by a resource group is `c` milliseconds, `r1` times of requests read `r2` KB data, `w1` times of write requests write `w2` KB data, and the number of non-witness TiKV nodes in the cluster is `n`. Then, the formula for the total RUs consumed by the resource group is as follows:
-
-`c`\* 1/3 + (`r1` \* 0.25 + `r2` \* 1/64) + (1.5 \* `w1` + `w2` \* 1 \* `n`)
+> **Note:**
+>
+> - Each write operation is eventually replicated to all replicas (by default TiKV has 3 replicas). Each replication operation is considered a different write operation.
```
Comment (qiancai) [inline tidb-resource-control.md:85]:
```suggestion
> - Each write operation is eventually replicated to all replicas (by default, TiKV has 3 replicas). Each replication operation is considered a different write operation.
```

File: tidb-resource-control.md [hunk 7]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
...
+> - In addition to queries executed by the user, RU can be consumed by background tasks, such as automated statistical information collection.
```
Comment (qiancai) [inline tidb-resource-control.md:86]:
```suggestion
> - In addition to queries executed by users, RU can be consumed by background tasks, such as automatic statistics collection.
```

File: tidb-resource-control.md [hunk 8]
```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
...
+> - The preceding table lists only the resources involved in computing RUs for on-premises TiDB clusters, excluding the network and storage components. For TiDB Serverless RUs, see [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/).
```
Comment (qiancai) [inline tidb-resource-control.md:87]:
```suggestion
> - The preceding table lists only the resources involved in RU calculation for TiDB Self-Hosted clusters, excluding the network and storage consumption. For TiDB Serverless RUs, see [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/).
```

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**